src/main/groovy/pl/allegro/tech/build/axion/release/domain/VersionConfig.groovy

```diff
@@ -148,6 +148,12 @@ class VersionConfig {
             resolvedVersion = getUncachedVersion()
         }
     }
+	
+	String getScmVersion() {
```

Comment:
why is this method here? i don't want to expose additional properties in `VersionConfig`